File: 1erdepartamental/python/7.1.identificar_tamanios/main.py
"""
Programa: Identificar Tamaños
Demuestra uso de sys.getsizeof() para analizar memoria
"""

from core.analizador import AnalizadorTamanios, Persona

if __name__ == "__main__":
    analizador = AnalizadorTamanios()
    analizador.analizar_tipos_basicos()
    analizador.analizar_strings()

    p = Persona("Hugo", 19)
    analizador.analizar_objeto(p)

File: 1erdepartamental/c++/7.1.identificar_tamanios/main.cpp
#include "core/AnalizadorTamanios.h"

int main() {
    // Crea el analizador y ejecuta analisys
    AnalizadorTamanios analizador;
    analizador.analizarTiposBasicos();
    analizador.analizarString();
    analizador.analizarObjeto();
    return 0;
}
